src/status_im/ui/screens/wallet/transactions/styles.cljs

```diff
@@ -51,6 +51,20 @@
    :margin-top        22
    :margin-horizontal 92})
 
+(defstyle amount-time
```

Comment:
`defstyle` is used for platform differences, 
```
(defstyle amount-time
 {:flex-direction  :row
  :justify-content :space-between
  :ios {:padding-top 13}
  :android {:padding-top 14}})
```
you don't need it for tx-amount and tx-time